### DISCUSSION TITLE
Add support to baijiayun

### DISCUSSION THREAD

**Discussion by JoyYUAN88** [wxx9248/baijiayunxiao-downloader#408] [Ideas]
能不能加个对百家云www.baijiayun.com网址上的ev1视频的下载和解密？原来这个baijiayunxiao和baijiayun都属于百家云的。

**Comment by wxx9248** [comment]
#188
#221

因为我没百家云的账号，我不能测试程序能不能直接用。我能做的事是加一个匹配地址，然后放到一个新分支里，然后如果你如果能帮忙测试一下就最好了。如果通过的话，我可以推一个patch到master
